views/email/a/v1.tx

```diff
@@ -1,5 +1,5 @@
                                      <p style="margin-bottom: 1em">Recently we found that <b>66%</b> of people who use Private Browsing (aka "Incognito Mode") over-estimate the protection it provides:</p>
-<p style="margin-bottom: 1em; padding-top: 10px; padding-left:10px"><img src="https://duckduckgo.com/assets/email/v1.png?atb=<: $subscriber.flow :>" style="width:100%; padding-left: 5px; padding-bottom: 10px; color: #333333; font-size: 14px; font-family: sans-serif; font-style:italic; background: #ffffff" alt="Disabled Image: Chart of misconceptions about Private Browsing Mode"/>
+<p style="margin-bottom: 1em; padding-top: 10px; padding-left:0;"><img src="https://duckduckgo.com/assets/email/v1.png?atb=<: $subscriber.flow :>" style="width:100%; padding-left: 5px; padding-bottom: 10px; color: #333333; font-size: 14px; font-family: sans-serif; font-style:italic; background: #ffffff" alt="Disabled Image: Chart of misconceptions about Private Browsing Mode"/>
```

Comment:
removes the left-padding to look as:

![screen shot 2017-04-19 at 16 27 13](https://cloud.githubusercontent.com/assets/1329127/25185217/1228be86-251d-11e7-8bf1-fd62ad23a542.png)